### DISCUSSION TITLE
Tables as in dxfwrite in ezdxf

### DISCUSSION THREAD

**Discussion by mpele** [mozman/ezdxf#282] [General]
Hi,

As I understand Tables are not supported yet in ezdxf and will not be soon. I am looking for some solution to provide a table on the newly created document and for me, it would be acceptable solution presented in dxfwrite ( https://dxfwrite.readthedocs.io/en/latest/entities/table.html#example ), but I wasn't able to find a corresponding example in ezdxf. 

Is it possible to create these tables with ezdxf?

Thanks in advance,

**Comment by mozman** [comment]
The dxfwrite example is [here](https://github.com/mozman/dxfwrite/blob/master/examples/table.py).

You can also use the undocumented table add-on for ezdxf, which is a port from dxfwrite, an example is [here](https://github.com/mozman/ezdxf/blob/master/examples/addons/using_table_addon.py).

**Reply by mpele** [reply]
Thank you. This undocumented example is what I needed!
